pkgs/development/python-modules/scitools/default.nix

```diff
@@ -0,0 +1,20 @@
+{lib, fetchurl, python, buildPythonPackage, numpy, matplotlib}:
+
+buildPythonPackage rec {
+  pname = "scitools";
+  version = "0.9.0";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
```

Comment:
This can be 'simplified' by using `fetchFromGitHub` instead:

``` nix
{
  src = fetchFromGitHub {
    owner = "hplgit";
    repo = "scitools";
    rev = "${name}";
    sha256 = "...";
  };
}
```

Just run `nix-prefetch-url --unpack 'https://github.com/hplgit/scitools/archive/scitools-0.9.0.tar.gz'` to get the sha256 (if it's different).